### ISSUE TITLE
Aria-label missing for Cypress tests

### ISSUE THREAD

**Issue by Zanda94** [cortex-js/cortexjs.io#34]
Good day @arnog - I  hope you are well. I have been using this library for a while now and have no complaints, it has really been outstanding, thank you. I want to find out if the aria-label ([aria-label="math input field"]) has been removed from the math-field, as I use this for Cypress tests, and it can no longer find it anymore. Is there an alternative solution I can try if it has been removed? I am currently using the aria label to find the 'input' field and 'paste' latex mathematics in it to test some of my personal work.

**Comment by Zanda94** [comment]
I can get it like that, but when I use the type API from Cypress, some of the LaTeX commands do not work properly- it keeps replacing the curly left brace with the command for it instead of it being part of the latex command being executed - for example, if I type \text{ - it converts this to  \text\left\lbrace\right} as input? Is there perhaps a workaround or something I need to do to stop that from happening?
